x/claim/keeper/msg_server_claim_eth.go

```diff
@@ -57,13 +57,28 @@ func (k msgServer) ClaimEth(goCtx context.Context, msg *types.MsgClaimEth) (*typ
 		Address:                msg.Creator,
 		Chain:                  types.ARKEO,
 		InitialClaimableAmount: ethClaim.InitialClaimableAmount,
-		ActionCompleted:        []bool{false, false},
+		ActionCompleted:        []bool{true, false, false}, // true so they do not make them claim again on arkeo.
```

Comment:
This make me to think we should flat out `ActionCompleted` to more concrete fields  like
```
ForeignChainClaim bool
Stake bool
Vote bool
```
By using `ActionComplete` as a slice , it is not easy to understand , I need to always trying to figure out what the first `true` means , second `false` means. etc.

Reply:
Cool , we could do that in a seperate PR , don't want to hold off this one for any longer